server/pbench/lib/config/pbench-server.cfg.example

```diff
@@ -4,6 +4,7 @@
 # An installation will want to override at least the following:
 mailto = admins@example.com
 default-host = pbench.example.com
+version=002
```

Comment:
Do we want to expose the version number as a configuration variable?

Reply:
No - deleted. The default config file will have a `versions = 001 002` def to indicate that the server handles both (it will also be needed in the installation to create the host-info-url files).